turtle_graphics/loop.js

```diff
@@ -0,0 +1,50 @@
+// Drawing a grid
```

Comment:
Remove unnecessary files if they are not being used. If you want to keep them as reference put them on a different branch